cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -32,7 +35,7 @@ function docker_desktop_version {
 
 
 if [[ ${PWD} != ${HOME}* ]]; then
-  printf "\n\nWARNING: Project should usually be in a subdirectory of the user's home directory.\nInstead it's in ${PWD}\n\n"
+  printf "\n\nWARNING: Project should usually be in a subdirectory of the user's home directory.\nInstead it's in %s\n\n" "${PWD}"
```

Comment:
Not sure why to make these printf changes, but they're fine.

Reply:
I followed this recommendation https://github.com/koalaman/shellcheck/wiki/SC2059